Packs/Syslog/Integrations/SyslogSender/SyslogSender.py

```diff
@@ -237,7 +237,7 @@ def init_manager(params: dict) -> SyslogManager:
     self_signed_certificate = params.get('self_signed_certificate', False)
     if not address:
         raise DemistoException('A address must be provided.')
-    if port and (port < 0 or MAX_PORT < port):
+    if port and (port < 0 or port > MAX_PORT):
```

Comment:
Please either undo this fix, or add a `documentation and metadata updates` line to the release note